src/main/java/edu/ucla/library/bucketeer/Config.java

```diff
@@ -81,6 +84,12 @@ public final class Config {
     // A configuration option for turning on or off feature flags
```

Comment:
I see you changed the `//` on line 24 to `/** */` -- could also do that here and for the other fields to make them Javadocs instead of comments.